README.md

```diff
@@ -58,37 +63,32 @@ windows.
 
 ### Subscribe to events
 
-Subscribing to events is also simple. The following example tells Chasqui to
-forward events to this subscriber's `my-app` queue. Chasqui will detect whether
-you are using Sidekiq or Resque and generate an appropriate worker class for you.
-Within the subscriber block, you use the `on(event_name)` method to register blocks
-of code to handle that event.
+Subscribing to events is also simple. In the following example, we create a
```

Comment:
@jbgo here I would avoid "simple" and just say what you are doing.  Also, I think it is better to show the code first then follow with the text.